src/runner/index.js

```diff
@@ -121,9 +125,11 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    run ({ skipJsErrors, quarantineMode } = {}) {
-        this.opts.skipJsErrors   = !!skipJsErrors;
-        this.opts.quarantineMode = !!quarantineMode;
+    run ({ skipJsErrors, quarantineMode, elementAvailabilityTimeout } = {}) {
+        this.opts.skipJsErrors               = !!skipJsErrors;
+        this.opts.quarantineMode             = !!quarantineMode;
+        this.opts.elementAvailabilityTimeout = elementAvailabilityTimeout === void 0 ?
```

Comment:
It's a public API option and we can get here anything (function, object, number). What we should do here, what the best practice? Maybe check if `typeof elementAvailabilityTimeout !== 'number'` set it to default value? Or raise an exception if it's not a nubmer? /cc @inikulin